lib/AcqCheckboxFilter/AcqCheckboxFilter.js

```diff
@@ -21,14 +21,10 @@ const AcqCheckboxFilter = ({
   onChange,
   options = [],
 }) => {
-  const dataOptions = options.map(option => (
-    'labelId' in option
-      ? {
-        label: <FormattedMessage id={option.labelId} />,
-        value: option.value,
-      }
-      : option
-  ));
+  const dataOptions = options.map(option => ({
+    ...option,
+    label: option.labelId ? <FormattedMessage id={option.labelId} /> : option.label,
```

Comment:
could you please check console warnings? In this case you'll pass `labelId` to `CheckboxFilter`, where it's absent in `propTypes`. I predict console warning there about not expected attribute https://github.com/folio-org/stripes-smart-components/blob/e7e464c1d6fb5c63d2cfad033a9ccfbb3e43ae6c/lib/SearchAndSort/components/CheckboxFilter/CheckboxFilter.js#L9

Reply:
There are no warnings about `labelId`

Reply:
so should we wait for them?) I don't think it's a good idea to pass down props, that are not expected there. We introduced `labelId` on our layer to pass down a proper `label` attribute, so no need to pollute options objects with unknown properties

Reply:
`labelId` was removed from `dataOptions`

Reply:
There is no need in the additional operations, please refer this code https://github.com/folio-org/stripes-acq-components/blob/master/lib/FieldSelect/FieldSelect.js#L17

Reply:
Fixed.`labelId` was renamed in `dataOptions` due to `AcqCheckboxFilter` has the same prop